pkg/handler/v1/provider/openstack.go

```diff
@@ -45,17 +56,17 @@ func OpenstackSizeEndpoint(seedsGetter provider.SeedsGetter, presetsProvider pro
 			return nil, fmt.Errorf("error getting dc: %v", err)
 		}
 
-		username, password, domain, tenant, tenantID, err := getOpenstackCredentials(userInfo, req.Credential, req.Username, req.Password, req.Domain, req.Tenant, req.TenantID, presetsProvider)
+		credential, err := getCredentials(userInfo, req, presetsProvider)
 		if err != nil {
-			return nil, fmt.Errorf("error getting OpenStack credentials: %v", err)
+			return nil, err
```

Comment:
I would keep the added info to the error, it's always useful for debugging. You could consider using error wrapping: https://blog.golang.org/go1.13-errors

Reply:
I have not removed the added info, it is here 
https://github.com/kubermatic/kubermatic/pull/6852/files#diff-1ea1ef7250db578e0ba92bef24d40f20707fb444fb51c91f5fa05b14745a6bc5R39